waf.go

```diff
@@ -416,6 +417,7 @@ func (w *Waf) NewTransaction(ctx context.Context) *Transaction {
 // note: this is not thread safe
 func (w *Waf) SetDebugLogPath(path string) error {
 	if path == "" {
+		w.Logger.SetOutput(io.Discard)
```

Comment:
I think this may have been a bug, when changing `SetDebugLogPath` below to use empty string, tests failed with nil reference exception

Reply:
Does io.Discard work with wasm?

Reply:
Yeah it's just some no-op functions in Go so no problem on wasm